scripts/suite.py

```diff
@@ -173,6 +173,9 @@
                               but can be overide if passed again.
                               This is important for tests involving random facet
                               (path ends with '$' operator).
+ --rerun-skip-known <bool>    Skip jobs with known failure patterns during rerun.
```

Comment:
In my test the option name is better without _rerun_ prefix and in command form, i.e. `--skip-known-failures`, and add similar comment as for `-R` like `to be used with --rerun only`. Referring to known patterns json is not clear what is it and where is it. I would probably suggest to add another option with which there can be provided the path to the the know-patterns json or yaml, defaults to some file bundled with teuthology package.